app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -152,10 +152,19 @@ def start_date_row(course)
     def status_row(application_choice)
       {
         key: 'Status',
-        value: render(ApplicationStatusTagComponent.new(application_choice: application_choice)),
+        value: status_row_value(application_choice),
       }
     end
 
+    def status_row_value(application_choice)
+      if ApplicationStatusTagComponent.new(application_choice: application_choice).status == 'application_not_sent'
+        render(ApplicationStatusTagComponent.new(application_choice: application_choice)) +
+          tag.p('Your application was not sent for this course because references were not given before the deadline.', class: 'govuk-body govuk-body govuk-!-margin-top-2')
```

Comment:
Did you consider making the extra `p` part of the `ApplicationStatusTagComponent`?

Reply:
I did not, but it definitely should be. You're right 👍